bonsai/BudgetDetails/BudgetDetails.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  BudgetDetails.swift
+//  bonsai
+//
+//  Created by hoang on 07.12.2021.
+//
+
+import SwiftUI
+
+struct BudgetDetails: View {
+    var body: some View {
+        /*
+         Need to update when navigation bar will be added
+         */
+        VStack {
+            Spacer()
+            VStack(alignment: .leading) {
+                Spacer()
+                BudgetNameView()
+                    .frame(height: 33)
+                    .padding(.leading, 8)
```

Comment:
it should have .horizontal padding instead of .leading because if text will be too long it won't have padding on the trailing side

Reply:
fixed